backend/app/routers/invites.py

```
@@ -198,6 +198,26 @@ async def accept_invite(
+                cohort_service = CohortService(db)
+                instances_created = cohort_service._create_simulation_instances_for_student(
+                    cohort_id=cohort.id,
+                    student_id=current_user.id
+                )
+                if instances_created > 0:
+                    logger.info(f"Created {instances_created} simulation instances for user {current_user.id} in cohort {cohort.id}")
+                
+                # Invalidate simulation instances cache so student sees them immediately
+                redis_manager.delete(f"student_instances:{current_user.id}:all:all")
+                redis_manager.delete(f"missing_instances_check:{current_user.id}")
+            except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)